docs/en/stack/ml/df-analytics/evaluatedf-api.asciidoc

```diff
@@ -99,3 +99,35 @@ closer to 1, the better the algorithm performance.
 The {evaluatedf-api} can return the false positive rate (`fpr`) and the true 
 positive rate (`tpr`) at the different threshold levels, so you can visualize 
 the algorithm performance by using these values.
+
+
+[discrete]
+[[ml-dfanalytics-regression-evaluation]]
+=== {regression-cap} evaluation
+
+This evaluation type is suitable for evaluating {regression} models. The 
+{regression} evaluation type offers the following metrics to evaluate the model 
+performance:
+
+* Mean squared error (MSE)
+* R-squared (R^2^)
+
+
+[discrete]
+[[ml-dfanalytics-mse]]
+==== Mean squared error
+
+You can measure how well the {reganalysis} model is performing by computing the 
+average squared sum of the difference between the true value and the value that 
+the {regression} model predicted. (Avg (predicted value-actual value)^2^).
+This value is called mean squared error.
+
+
+[discrete]
+[[ml-dfanalytics-r-sqared]]
+==== R-squared
+
+Another evaluation metrics for {reganalysis} is R-squared (R^2^). It measures 
+the perfection of the fit of the predicted values. The value of R^2^ ranges from 
+0 to 1, where 1 is the perfect fit. For example, if the R^2^is 0.50, then 50% of 
+the values are predicted correctly by the model.
```

Comment:
This isn't quite right. R^2 is a measure of goodness of fit and it relates to how much of the variance in a metric quantity is explained. A value of 1 indicates the predictions explain all the variance in the data, i.e. prediction_i = actual_i for all i. A value of 0.5 indicates that variance actual_i - predicted_i is half the variance of actual_i. This wiki page gives quite a lot of information: https://en.wikipedia.org/wiki/Coefficient_of_determination. I'd consider something like the following:

Another evaluation metrics for {reganalysis} is R-squared (R^2^). It measures how much of the variation in the data the predictions are able to explain. The value of R^2^ is less than or equal to 1, where 1 indicates that the predictions and true values are equal, a value of 0 is obtained for a predictor which is the mean of the actual values. A value of 0.5 for R^2^ would indicate that, on average, the predictions are about 30%, or 1 - 0.5^(1/2)^, closer to true values than their mean.